docs/advanced-topics/verification.rst

```diff
@@ -0,0 +1,126 @@
+Verifying a test parameter file
+===============================
+
+1.  Go to
+    https://dockstore.org/static/swagger-ui/index.html#/extendedGA4GH/toolsIdVersionsVersionIdTypeTestsPost
+2.  Click “Try it out”
+3.  Provide a “type”. See the
+    description for allowable values.
+4.  Provide the TRS ID for the tool/workflow being verified.
+    For example, the `dockstore-tool-md5sum`_ tool has the TRS ID:
+    “quay.io/briandoconnor/dockstore-tool-md5sum” as shown in the “Info”
+    tab with the label: “TRS CWL” or “TRS WDL”
+5.  Provide the version_id of the tool/workflow to verify. It can be any
+    version listed in the Version tab of the tool/workflow.
+    `dockstore-tool-md5sum <https://dockstore.org/containers/quay.io/briandoconnor/dockstore-tool-md5sum:master?tab=versions>`__
+    has the following versions currently: 1.0.4, master, develop, 1.0.3,
+    latest, 1.0.2, 1.0.1, and 1.0.0.
+6.  Provide the “relative_path” of the test parameter file being
+    verified. The path of the test parameter file is relative to the
+    primary descriptor. This path can be found using the `files
+    endpoint`_ or by viewing the files tab of a tool/workflow such as:
+    https://dockstore.org/containers/quay.io/briandoconnor/dockstore-tool-md5sum:1.0.4?tab=files
+    and then further selecting the Test Parameter Files tab and view the
+    right-most “File” dropdown. This relative path must be a test
+    parameter file, providing a descriptor will not work.
+7.  Provide the “platform”. Some examples are: HCA, Cromwell, Arvados,
+    etc.
+8.  Select the “verified” status either as “true” or “false”. Use “true”
+    to verify, “false” to “unverify”.
+9.  Provide “metadata”, this is typically the verifier’s identity which
+    can be something like “GA4GH/DREAM Challenge”
+10. Lastly, provide your Dockstore token using the lock icon at the top
+    right of the endpoint
+
+Below is a screenshot of someone verifying the “test.json” test
+parameter file of the “master” version of the “dockstore-tool-md5sum”
+tool.
+
+.. figure:: /assets/images/docs/verification/swagger.png
+   :alt: Swagger-UI
+
+   Swagger-UI
+
+The curl command results in something like:
+
+::
+
+   curl -X POST "https://dockstore.org/api/api/ga4gh/v2/extended/quay.io%2Fbriandoconnor%2Fdockstore-tool-md5sum/versions/master/CWL/tests/test.json?platform=Dockstore%20CLI&verified=true&metadata=Phase%201%20GA4GH%20Tool%20Execution%20Challenge" -H  "accept: application/json" -H  "Authorization: Bearer iamafakebearertoken"
+
+A successful response will result in something like:
+
+::
+
+   {
+     "Dockstore CLI": {
+       "metadata": "Phase 1 GA4GH Tool Execution Challenge",
+       "verified": true
+     }
+   }
+
+Viewing on Dockstore.org
+========================
+
+There are 3 new indicators in Dockstore.org that indicates whether or
```

Comment:
This part is useful for users that aren't admins or curators, maybe order this first before the section above

Reply:
Putting this above causes this tutorial on verification to not flow properly. For users that aren't admins or curators, they shouldn't even read this document.

Reply:
I rather meant that it might make sense to re-order and add text to make it flow properly. 

Right now, it seems that we don't have docs on what it means to be verified on a particular platform and this might be a good opportunity to write something that can be targeted at workflow developers/platforms (i.e. "if you can get your workflow tested on multiple platforms this is how we can show it off" or "if you're a platform owner, this is how we can advertise that many workflows work on your platform"

Also, might be a good place to describe the tooltester logs feature

Reply:
I think this page is for admins/curators, and the FAQ entry is for normal users. So I prefer the current order.